packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -293,7 +471,7 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
               },
             });
 
-            const inputProps = getInputProps({
+            const inputProps: object = getInputProps({
```

Comment:
I don't have a clue as to why, but ts computes the return type to `never`… but this cast seems good enough™ since we don't really care about the precise shape.